charts/pravega-operator/Chart.yaml

```diff
@@ -1,12 +1,9 @@
+apiVersion: v2
 name: pravega-operator
-version: 0.5.0
-appVersion: 0.5.0
-description: pravega operator Helm chart for Kubernetes
+description: Pravega Operator Helm chart for Kubernetes
+version: 0.1.0
```

Comment:
Given that this will be the first time we will be publishing our charts, shouldn't we be starting with the version `0.1.0`

Reply:
Fair enough -- just verifying it wasn't an oversight.
What happens when we get to `0.5.0` (which is the current -- albeit unpublished officially -- version)?

(this comment doesn't apply to the other chart of course since _that_ chart was never published unofficially either)